environments/servlet/pom.xml

```diff
@@ -152,6 +153,13 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>jakarta.websocket</groupId>
+                <artifactId>jakarta.websocket-api</artifactId>
```

Comment:
I don't know why we need this dependency now, it must have been part of other API and then separated?